src/main/java/com/example/demo/repository/MockedBookRepositoryImpl.java

```diff
@@ -0,0 +1,49 @@
+package com.example.demo.repository;
+
+import com.example.demo.model.Book;
+import com.example.demo.model.util.Language;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class MockedBookRepositoryImpl implements BookRepository {
+
+    private ArrayList<Book> books;
```

Comment:
To jest List<Book>. W takich przypadkach korzystamy z interfejsow o ile to mozliwe, potem latwiej wymienic ArrayListe na jakas inna